proposals/save_ml_model.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "save_ml_model",
+    "summary": "Save a ML model",
+    "description": "Saves a machine learning model as part of a batch job.\n\nThe model will be accompanied by a separate STAC Item that implements the [ml-model extension](https://github.com/stac-extensions/ml-model).",
```

Comment:
> The model will be accompanied by a separate STAC Item that

What does "accompanied" practically mean? Should there be an additional job result asset? Or should this be an job result link item?

The reason I'm asking is that we want to streamline the detection of the model's URL at the client side.

e.g. see https://github.com/Open-EO/openeo-python-client/pull/576 we we currently have a highly implementation-specific hack
```
ml_model_metadata_url = [
    link 
    for link in links if 'ml_model_metadata.json' in link['href']
][0]['href']
```

Reply:
Good question, I guess we should clarify that.

On the other hand, please note that this PR is implicitly outdated as the ML Model extension in STAC is likely going to be replaced by another extension. So this generally needs more work (which I have no plans to do anytime soon).

Reply:
https://github.com/crim-ca/mlm-extension